### PR TITLE
Implement new styling to recipe list and recipe view

### PR DESCRIPTION
**Pull request by jonasolof**

### CODE REVIEW

File: src/components/RecipeList.tsx [hunk 1]
```diff
@@ -91,42 +94,63 @@ const RecipeList: React.FC<RecipeListScreenProps> = ({ navigation }) => {
...
+    justifyContent: 'space-between',
+    alignItems: 'center',
   },
   listItem: {
+    borderColor: '#666666',
+    borderWidth: 1,
     flexDirection: 'row',
-    marginBottom: 8,
+    marginBottom: 16,
+    // minHeight: 80,
```
Comment (saarikt1) [inline src/components/RecipeList.tsx:142]: Remove commented-out lines, if there's no clear reason for them.

File: src/styles/TextStyles.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  headingLarge: {
+    fontFamily: 'LibreFranklin_700Bold',
+    fontSize: 32,
+    lineHeight: 36,
+  },
+  headingSmall: {
+    fontFamily: 'LibreFranklin_700Bold',
+    fontSize: 20,
+    lineHeight: 22,
+    // marginBottom: 4,
```
Comment (saarikt1) [inline src/styles/TextStyles.tsx:13]: Also here.

File: App.tsx [hunk 3]
```diff
@@ -1,6 +1,15 @@
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
 import { SafeAreaProvider } from 'react-native-safe-area-context';
+
+import AppLoading from 'expo-app-loading';
+
+import {
+  useFonts,
+  LibreFranklin_400Regular,
```
[the remark this answers is not in the document]
Comment (jonasolof) [reply App.tsx]: This makes a lot of sense, implemented in one of my commits

File: App.tsx [hunk 4]
```diff
@@ -12,22 +21,31 @@ export type RootStackParamList = {
...
-      </NavigationContainer>
-    </SafeAreaProvider>
-  );
+  let [fontsLoaded] = useFonts({
+    LibreFranklin_400Regular,
+    LibreFranklin_700Bold,
+  });
+  if (!fontsLoaded) {
+    return <AppLoading />;
+  } else {
```
[the remark this answers is not in the document]
Comment (jonasolof) [reply App.tsx]: Installed ESLint and fixed all of those issues. Thanks again

File: src/components/RecipeList.tsx (hunk 1 above)
Comment (jonasolof) [reply src/components/RecipeList.tsx]: ✅

**Review by saarikt1 — APPROVED** [review]
Looks good to me now. I think the [WIP] can be edited out from the name at the moment you feel like the PR is good to merge, and then I can just check that everything is ok and do the merge 👍

**Pull request merged**